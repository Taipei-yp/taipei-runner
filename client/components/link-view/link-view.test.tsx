import React from "react";
import { MemoryRouter as Router } from "react-router-dom";
import TestRenderer from "react-test-renderer";
import { LinkView } from ".";

describe("Component LinkView", () => {
  const linkview = TestRenderer.create(
    <Router>
      <LinkView to="/test" label="testlabel" className="testclass" size="l" />
    </Router>,
  );
  it("correct render", () => {
    expect(linkview.toJSON()).toMatchSnapshot();
  });
});
