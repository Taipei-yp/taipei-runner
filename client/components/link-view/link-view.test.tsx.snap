// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component LinkView correct render 1`] = `
<a
  className={
    Object {
      "has": [Function],
      "is": [Function],
      "mix": [Function],
      "split": [Function],
      "state": [Function],
      "toString": [Function],
    }
  }
  href="/test"
  onClick={[Function]}
>
  testlabel
</a>
`;
